src/common/resources.rs

```diff
@@ -39,22 +40,30 @@ impl CpuRequest {
     }
 }
 
-#[derive(Serialize, Deserialize, Debug, Clone, Hash, Eq, PartialEq)]
+pub type TimeRequest = Duration;
+
+#[derive(Default, Serialize, Deserialize, Debug, Clone, Hash, Eq, PartialEq)]
 pub struct ResourceRequest {
     cpus: CpuRequest,
+
+    #[serde(default)]
+    time: TimeRequest, // Minimal remaining time of the worker life time needed to START the task
```

Comment:
Documentation should be above the field and should start with `///`, this formatting is quite weird.